swarm/network/kademlia.go

```diff
@@ -277,7 +275,7 @@ func (k *Kademlia) On(p *Peer) (uint8, bool) {
 	}
 	log.Trace(k.string())
 	// calculate if depth of saturation changed
-	depth := uint8(k.saturation(k.MinBinSize))
+	depth := uint8(k.saturation())
```

Comment:
return uint8 from `saturation`?

Reply:
I don't think so, because bins are `int` everywhere else. @nonsense raised the question whether bin should be its own type. We could consider this, but then let's do it in one PR and change it everywhere.